frontend/src/views/CanteenEditor/CanteenForm.vue

```diff
@@ -473,6 +482,23 @@ export default {
         if (Object.prototype.hasOwnProperty.call(payload, x) && payload[x] === "") payload[x] = null
       })
 
+      if (!bypassTechnicalControl) {
+        if (this.canteen.productionType === "central_serving" && this.canteen.satelliteCanteensCount == 1) {
+          this.displayTechnicalControlDialog("Etes-vous sûr de ne livrer qu'un seul autre site de service ?")
+          return
+        }
+        const isCentralCanteen =
+          this.canteen.productionType === "central_serving" || this.canteen.productionType === "central"
+        if (isCentralCanteen && parseInt(this.canteen.satelliteCanteensCount) >= 250) {
+          this.displayTechnicalControlDialog(
+            `Vous êtes sur le point de déclarer une livraison depuis votre cuisine centrale à ${parseInt(
+              this.canteen.satelliteCanteensCount
+            )} établissements de service. Êtes-vous sûr de vouloir continuer ?`
```

Comment:
```suggestion
            )} établissements de service. Voulez-vous continuer ?`
```
Pour éviter du langage genré